numero/numero.cpp

```diff
@@ -77,41 +77,38 @@ void load_time_zone(std::filesystem::path& exe) {
  *  Prepares for the computation by parsing the options and doing initial setup
  */
 std::ifstream prepare(int argc, char** argv) {
-	const char *OPT_HELP = "h", *OPT_OUTPUT_LINES = "n", *OPT_TIME_ZONE_ID = "tz", *OPT_INPUT_FILE = "input-file";
+	const std::string OPT_HELP{"help"}, OPT_OUTPUT_LINES{"output-lines"}, OPT_TIME_ZONE_ID{"time-zone"}, OPT_INPUT_FILE{"input-file"};
```

Comment:
nit: maybe these would be a better formatted as a list.

Reply:
I am not fully sure what you mean by a list, but did some changes that I think are along the lines you mention